Makefile

```diff
@@ -13,10 +13,16 @@ include Makefile.global
 
 all: extension
 
+
+#build columnar only
+columnar:
+	$(MAKE) -C src/backend/columnar all
 # build extension
-extension: $(citus_top_builddir)/src/include/citus_version.h
+extension: $(citus_top_builddir)/src/include/citus_version.h columnar
 	$(MAKE) -C src/backend/distributed/ all
-install-extension: extension
+install-columnar: columnar
```

Comment:
maybe

```suggestion
install-columnar-extension: columnar
```

Reply:
I think `install-columnar` is verbose enough unless you see some ambiguity.